### PR TITLE
Feature/actions

### PR DESCRIPTION
**Pull request by shahabl**
Adding support for Action buttons on Android & Android Wear, including support for voice input.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -417,6 +417,57 @@ window.plugin.notification.local.add({ sound: '/www/audio/beep.mp3' });
...
+            voice:{            // use this to ask user for input
+                label:"Got Milk",
+                choices:['yes', 'no', 'never!'],
+                freeform:false // shows choices on wearable, not taking voice input
+            }
+        }
+    ]
+});
+
+window.plugin.notification.local.onclick = function(id, state, action, json) {
```
Comment (ptbrowne) [inline README.md]: you did not put the input argument here

**Comment by vcalvello** [comment]
hey guys, this is a nice feature (actually what I need right now :)
anyone to review it and merge?

**Comment by katzer** [comment]
@shahabl cc @vcalvello - First of all thanks a lot for your PR. These feature is very useful and I want to support that!

There are multiple forks and PR who dress these feature. I need to get an overview first.

**Comment by katzer** [comment]
@shahabl Actions have been added. I will take a look to support wearables and voice input which is part of your PR.

**Pull request closed without merging**